lib/schema_associations/active_record/associations.rb

```diff
@@ -90,6 +106,11 @@ def _load_schema_associations_associations
             _define_association(:belongs_to, foreign_key)
           end
 
+          _deferred_associations.each do |a|
+            argstr = a[:args].inspect[1...-1] + ' # deferred association'
```

Comment:
Sorry, where did this come from?

Reply:
_define_association sets it when it does the call to _create_association (which is used for the debug output of what associations were created).

Reply:
I'm a little concerned that we're relying on an implementation detail here, I'd prefer to punt this to as late as possible, perhaps as something that `create_association` does. Ohwell, there's already other code that does similar things 🤷‍♂️